### DISCUSSION TITLE
Enquiry about flet_webview

### DISCUSSION THREAD

**Discussion by dsandms** [flet-dev/flet#2805] [General]
We were anxiously awaiting the webview implementation for web browser. Do we have any estimate. Last feedback was in Flet 1.0 targeted in Jan 2024. But since things have changed now and dates are May 24 so we just wanted to inquire on status for Webview and timelines for availability please. Appreciate your kind and continued support.

**Comment by ndonkoHenri** [comment]
Webview is already available: https://flet.dev/docs/controls/webview/

However, it's current implementation is mobile-only.
You can subscribe to [this](https://github.com/flet-dev/flet/issues/432) webview issue to get updates when it get's improved and made available for other platforms - web and desktop.
